### PR TITLE
feat(pattern): add progress pattern

### PR DESCRIPTION
**Pull request by yhuard**

### CODE REVIEW

File: src/framework/molecules/progress/templates/progress.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+src:
+  styles:
+    - '../styles/progress.less'
+---
+<div class="farnet-progress porgress">
+  <div class="progress-bar progress-bar-success" role="progressbar" aria-valuenow="20" aria-valuemin="0" aria-valuemax="100" style="width: 20%">
```
Comment (kalinchernev) [inline src/framework/molecules/progress/templates/progress.html]: Is there a reason the `progress-bar` and `progress-bar-success` are not `progress__bar` and `is-active` or `is-successful`?

File: src/framework/organisms/flag-strategy/scripts/flag-strategy.js [hunk 2]
```diff
@@ -1,9 +0,0 @@
-export function flagStrategy(selector) {
```
Comment (kalinchernev) [inline src/framework/organisms/flag-strategy/scripts/flag-strategy.js]: I guess the implementation has something that covers for this?
I see no other reason the javascript part would be problematic [having fallback](https://getmdl.io/components/index.html#loading-section).

File: src/framework/molecules/progress/templates/progress.html (hunk 1 above)
Comment (yhuard) [inline src/framework/molecules/progress/templates/progress.html]: All the `progress-*` classes are Bootstrap classes. That's the only reason

File: src/framework/organisms/flag-strategy/scripts/flag-strategy.js (hunk 2 above)
Comment (yhuard) [inline src/framework/organisms/flag-strategy/scripts/flag-strategy.js]: The thing is, it's none of my business to care about how the integrators will build their template. All I need to know is that those values are server-side rendered and not dynamic, so we don't need any JS script to update the component.

**Pull request merged**